requirements/pytorch/strategies.txt

```diff
@@ -1,3 +1,6 @@
+# NOTE: the upper bound for package version is only set for CI stability and it is dropped while installing this package
```

Comment:
IMO this should be mentioned somewhere like CONTRIBUTING.md or a README.md inside the `requirements` directory

Reply:
We can add it but still worse to remind in the file, it is easy tor forget...